src/extensions/extensionInterface.ts

```diff
@@ -124,18 +156,26 @@ export const sortingCurationReducer: Reducer<SortingCuration, SortingCurationAct
         }
         else return state
     }
+    else if (action.type === 'MergeUnits') {
+        return {
+            ...state,
+            mergeGroups: simplifyMergeGroups([...(state.mergeGroups || []), action.unitIds])
+        }
+    }
+    else if (action.type === 'UnmergeUnits') {
+        return {
+            ...state,
+            mergeGroups: simplifyMergeGroups((state.mergeGroups || []).map(g => (g.filter(x => (!action.unitIds.includes(x))))))
```

Comment:
To unmerge units, we just need to remove the units from everywhere they appear in the merge groups, and then simplify.

Reply:
It's not immediately obvious what the intended behavior of the `UnmergeUnits` verb is.

Let's assume that we never get here without having a simplified (disjoint) merge group list as I think that'll be ensured by the way the `MergeUnit` verb is defined.

So, supposing I have merge groups like: `[[1, 2, 3, 5], [4, 6, 7], [8, 10]]`.

- If I call `UnmergeUnits` with `[1]`, I should get back `[[2, 3, 5], [4, 6, 7], [8, 10]]`, right? Or does it dissolve the group containing `1` entirely?
- If I call with `[1, 2]`, assuming the first merge group becomes `[3, 5]`--do `[1, 2]` remain grouped together?
- If I call with `[3, 4]`, which are not in the same group, do I wind up with `[[1, 2, 5], [6, 7]...]` having removed these items from their respective groups? Or does the unmerge fail, because units 3 and 4 weren't merged to each other?

I think it'd be helpful to document the intended behavior in comments here because we could get into some confusing situations.

Reply:
(Even if the UI looks like it disallows some of these situations from occurring)

Reply:
The behavior is: remove unit(s) from all merge groups and then simplify. Adding an issue to document more clearly.